UnityProject/Assets/Scripts/UI/Items/PDA/GUI_PDAUplinkItem.cs

```diff
@@ -24,10 +24,16 @@ public void OnPageDeactivated()
 		public void GenerateEntries(UplinkCategory category)
 		{
 			controller.SetBreadcrumb($"{controller.UPLINK_DIRECTORY}/categories/{category.CategoryName}/");
-			dynamicList.AddItems(category.ItemList.Count);
+			bool isNukie = controller.mainController.PDA.IsNukeOps;
+			int skipped = 0;
 			for (int i = 0; i < category.ItemList.Count; i++)
 			{
-				dynamicList.Entries[i].GetComponent<GUI_PDAUplinkItemTemplate>().ReInit(category.ItemList[i]);
+				if (isNukie || category.ItemList[i].IsNukeOps == false)
+				{
+					dynamicList.AddItem();
+					dynamicList.Entries[i].GetComponent<GUI_PDAUplinkItemTemplate>().ReInit(category.ItemList[i]);
+					skipped++;
```

Comment:
I don't think you're using this value anywhere

Reply:
ah bugger, your right
meant to remove that but forgot